### PR TITLE
RHIROS-1345 Probes for processor

### PR DESCRIPTION
**Pull request by saltgen**

### CODE REVIEW

File: clowdapp.yaml [hunk 1]
```diff
@@ -28,6 +28,26 @@ objects:
           limits:
             cpu: ${CPU_LIMIT}
             memory: ${MEMORY_LIMIT}
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
```
Comment (patilsuraj767) [inline clowdapp.yaml:34]: Here it should be `/metrics` and not `/`. Ref - https://github.com/RedHatInsights/ros-ocp-backend/blob/main/internal/utils/utils.go#L183

**Review by patilsuraj767 — CHANGES_REQUESTED** [review]

**Comment by kgaikwad** [comment]
\retest

File: clowdapp.yaml (hunk 1 above)
Comment (saltgen) [reply clowdapp.yaml]: Thanks @patilsuraj767, change pushed, do the the config params look fine?

**Review by patilsuraj767 — APPROVED** [review]
LGTM 👍

File: clowdapp.yaml (hunk 1 above)
Comment (patilsuraj767) [reply clowdapp.yaml]: Config look good for now, Later if we think that there are many http requests coming to the pod we can increase the `periodSeconds` time.

**Pull request merged**